lib/big_query/base/timestamp.ex

```diff
@@ -41,17 +41,20 @@ defmodule BigQuery.Base.Timestamp do
     Regex.replace(~r/[ \n\r\t]+/, str, " ")
   end
 
-  def group({data, meta}, intv, group_by) do
+  def group({data, meta}, intv) do
     data = intv.rollup.range(intv.from, intv.to, false)
-           |> Enum.map(&({&1, find_times(&1, data, group_by)}))
```

Comment:
This call was going real slow for large result sets.  I could have done some optimizations here (this was probably the most inefficient way I could have grouped the resultset by "time").  But since BQ is already good at grouping, I opted to make it do the fancy work.